test/unit/runnable.spec.js

```diff
@@ -670,6 +670,16 @@ describe('Runnable(title, fn)', function() {
         });
       });
     });
+
+    describe('when fn is not a function', function() {
+      it('should throw an error', function() {
+        var runnable = new Runnable('foo', 4);
+
+        runnable.run(function(err) {
+          expect(err.message, 'not to match', /fn\.call/);
```

Comment:
I think i'd rather it tested for the new message rather than lack of old one.

Reply:
Would you rather the match be the error message verbatim, or only a portion of?

Reply:
Changed to match the error message verbatim.